game/src/main/kotlin/content/area/misthalin/lumbridge/Victoria.kts

```diff
@@ -0,0 +1,53 @@
+package content.area.misthalin.lumbridge
+
+import content.entity.player.dialogue.*
+import content.entity.player.dialogue.type.*
+import content.entity.player.dialogue.type.choice
+import content.entity.player.dialogue.type.npc
+import content.entity.player.dialogue.type.player
+import world.gregs.voidps.engine.entity.character.npc.npcOperate
+import world.gregs.voidps.engine.entity.character.player.Player
+import world.gregs.voidps.engine.suspend.SuspendableContext
+
+npcOperate("Talk-to", "victoria*") {
```

Comment:
```suggestion
npcOperate("Talk-to", "victoria") {
```

`*` isn't needed as there aren't other victoria's

(it's the equivalent of `id. startsWith("Victoria")`)